message.go

```diff
@@ -215,6 +216,9 @@ func EnsureMessageService(m *Message) error {
 		if ok {
 			if allocIdRegex.MatchString(container_name) {
 				service = container_name[0:allocIdRegex.FindStringIndex(container_name)[0]]
+				if taskNumberRegex.MatchString(service) {
+					service = service[0:taskNumberRegex.FindStringIndex(service)[0]]
+				}
```

Comment:
should we also log task ID to distinguish those somehow within a single allocation?